### PR TITLE
Change CDN links

### PR DESCRIPTION
**Pull request by LukasDrgon**
Given the advantages of jsDelivr discussed in #1238 I think it would be a good idea to use it in the README as well. What do you think?

### CODE REVIEW

**Comment by ylemkimon** [comment]
I'm +1.

**Comment by ronkok** [comment]
Me too. Consistency is good of course. But more importantly, the existing README points to a CDN that takes 2 or 3 days to fully propagate the files.  It's common during those days to get a question on Gitter asking why the release doesn't work.

**Comment by edemaine** [comment]
Sounds good to me too. Thanks for the PR! 

Before merging, we might wait for @kevinbarabash to return, as he usually runs the release scripts. I wouldn't want to mess those up (though this PR seems to address them).

File: update-sri.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ Promise.all(process.argv.slice(3).map(file =>
             return pre + version + post;
         });
         return Promise.all(Object.keys(hashes).map(hash =>
-            read(path.join("dist", hashes[hash].file), null)
+            read(hashes[hash].file, null)
```
Comment (kevinbarabash) [inline update-sri.js]: why did this change?

Comment (MartinKolarik) [reply update-sri.js]: jsDelivr links already include `dist`, see above in readme

Comment (kevinbarabash) [reply update-sri.js]: Cool.  I double checked the regex on regex101.com and it checks out.  The regex could probably be simplified, but your changes are minimal so it's okay.

**Pull request merged**